SoomlaiOSProfile/SoomlaSerializableObject.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Eugene Butusov on 21/11/15.
+// Copyright (c) 2015 Soomla. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "UserProfileUtils.h"
+
+
+@interface SoomlaSerializableObject : NSObject {
```

Comment:
@borisspektor, oops, I did already implemented thing: https://github.com/soomla/soomla-ios-core/blob/master/SoomlaiOSCore/SoomlaEntity.h
I mean that it should be such in other Soomla modules (serializable things should be inherited from `SoomlaEntity`).